.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
```

Comment:
The rule 'strict' has been removed without providing an alternative or a reason for the removal.